RFCs/0009-Fn::ForEach.md

```diff
@@ -0,0 +1,1461 @@
+# Looping functionality for CFN Templates
```

Comment:
I'm happy with `Fn::ForEach` as a name, but I'd prefer to stay away from "loop" as much as possible, if there's a way to do that. At least use it as an opportunity to help users understand that they're using it in a functional programming way, rather than an imperative one.

Reply:
That makes sense! I've updated the title to not mention "looping", to ensure it isn’t misunderstood as being an imperative solution.